www/scipylib/donations.rst

```diff
@@ -57,6 +57,223 @@ infrastructure:
 *This list is ordered by time (most recent contributions first) and was last
 updated in April 2019.*
 
+
+Thanks
+------
+
+SciPy is a community project sponsored by Enthought, Inc.
+SciPy originated with code contributions by Travis Oliphant, Pearu
+Peterson, and Eric Jones.  Travis Oliphant and Eric Jones each contributed
+about half the initial code.  Pearu Peterson developed f2py, which is the
+integral to wrapping the many Fortran libraries used in SciPy.
+
+Below is a partial list of contributors to SciPy.  If you've
+been left off, please email the `SciPy Developers List`_.
+
+The SciPy development team would like to thank the following individuals for
+submitting issues, bugfixes, enhancements, and development infrastructure:
+
```

Comment:
How about we merge the `authors.py` feature first, and then we just make this a complete list for the whole project history?

Reply:
> How about we merge the `authors.py` feature first, and then we just make this a complete list for the whole project history?

Sounds good to me. For the time being I'll leave the file in this PR "as is"